src/main/kotlin/interpreter/stack/UserCodeStackFrame.kt

```diff
@@ -231,18 +323,73 @@ class UserCodeStackFrame(val function: PyUserFunction) : StackFrame() {
      * MAKE_FUNCTION.
      */
     fun makeFunction(arg: Byte) {
+        // toInt to use bitwise `and`
+        val flags = arg.toInt()
         val qualifiedName = stack.pop()
         require(qualifiedName is PyString) { "Function qualified name was not string!" }
 
         val code = stack.pop()
         require(code is PyCodeObject) { "Function code was not a code object!" }
-
+        if (flags and 8 == 8) {
+            val freevarCellsTuple = stack.pop()
+        }
+        if (flags and 4 == 4) {
+            val annotationDict = stack.pop()
+        }
+        if (flags and 2 == 2) {
+            val kwOnlyParamDefaultDict = stack.pop()
+        }
+        if (flags and 1 == 1) {
+            val positionalParamDefaultTuple = stack.pop()
+        }
         val function = PyUserFunction(code.wrappedCodeObject)
-        function.module = this.function.module
+        function.module = function.module
         stack.push(function)
         bytecodePointer += 1
     }
 
+    // Imports
+    /**
+     * IMPORT_NAME
+     */
+    fun importName(arg: Byte) {
+        // TODO: Do import
+        val fromList = stack.pop()
+        val level = stack.pop()
+        val name = function.code.names[arg.toInt()]
+
+        // TODO: Create module and push to stack
+        stack.push(PyNone)
+        bytecodePointer += 1
+    }
+
+    /**
+     * IMPORT_FROM
+     */
+    fun importFrom(arg: Byte) {
+        val module = stack.last
+        val attrName = function.code.names[arg.toInt()]
+        val attr = module.pyGetAttribute(attrName)
+        stack.push(attr)
+        bytecodePointer += 1
+    }
+
+    /**
+     * IMPORT_STAR
+     */
+    fun importStar(arg: Byte) {
+        val module = stack.pop()
+
+        // Assuming internalDict is __dir__
```

Comment:
`pyDefaultDir()` was added in methods and attributes v3 PR, which should be what you want for now.